### DISCUSSION TITLE
Thank you for your contributions (September 2023)

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11344] [Community]
Not long after the previous message, we'd again like to acknowledge all those who have generously volunteered their time and energy to support offline-first educational technology by contributing to Learning Equality's repositories.

In September, we received great work from @GarvitSinghal47, @Akila-I, @nikkuAg, @Jaspreet-singh-1032, @pwithnall, @dylanmccall, @muditchoudhary, @akash5100, @thesujai, @ShivangRawat30, and @yash1378. Thank you!

As usual, we're planning to mention all of you in our next [Community Newsletter](https://learningequalitynewsletter.getresponsepages.com/).

I'd like to use this opportunity to invite you one last time to our open-source community survey (https://github.com/learningequality/kolibri/discussions/11320). There, alongside other questions, we're also trying to find out what types of acknowledgment, in addition to the Newsletter and this message, would be supportive for your life and career. Even though volunteering for open-source projects can be a rewarding experience in itself, community is at the center of our work and vision, and we believe it is important to explore ways towards mutually beneficial collaboration.

**Comment by 0xV0YD** [comment]
@MisRob hello mam i would like to contribute now again as the time i was off , i learned about django , app dev and now i am ready to indulge fully in open source contribution can i start from setting up the app locally on my pc ?

**Reply by MisRob** [reply]
Hi @yash1378, I linked contributing guidelines in the other thread you posted.

**Reply by 0xV0YD** [reply]
got it mam
